docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -326,6 +326,34 @@ this table:
 
 No other types are supported.
 
+## Delta Lake table features
+
+The connector supports the following Delta Lake [table features](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#table-features):
```

Comment:
We may want to use https://docs.delta.io/latest/versioning.html though append-only isn't listed in the page.